app/src/main/java/com/github/mik629/android/fundamentals/ui/utils/Utils.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.mik629.android.fundamentals.ui.utils
+
+import android.content.Context
+import androidx.core.content.ContextCompat
+import com.github.mik629.android.fundamentals.R
+import com.github.mik629.android.fundamentals.databinding.RatingBarBinding
+import com.github.mik629.android.fundamentals.databinding.RatingBarItemBinding
+
+@Suppress("NOTHING_TO_INLINE")
+inline fun RatingBarBinding.setRating(context: Context, value: Float) {
+    val grey = ContextCompat.getColor(context, R.color.grey)
+    with(this) {
+        arrayOf(starOne, starTwo, starThree, starFour, starFive)
+            .forEachIndexed { index, star ->
+                star.setGreyIf(grey, value, index)
+            }
+    }
+}
+
+@Suppress("NOTHING_TO_INLINE")
+inline fun RatingBarItemBinding.setGreyIf(
+    color: Int,
+    rating: Float,
+    starIndex: Int
+) {
+    if (rating < starIndex) {
```

Comment:
think about recycler view

same ViewHolder is reused(view is reused)
first item has rating 0
second item has rating 5
after first item bind : all stars are GRAY
after second item bind: all stars still are GRAY

Reply:
Created an [issue](https://github.com/mik629/AndroidFundamentalsApp/issues/8) so it won't get los